README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.com/smudgelang/smear.svg?branch=master)](https://travis-ci.com/smudgelang/smear)
```

Comment:
The [given badge](https://api.travis-ci.com/smudgelang/smear.svg?branch=master) just says "build" and that is the only one Travis supplies.  I had found one when I was doing this for smudge where you can set the text, [like this](https://img.shields.io/travis/smudgelang/smear.svg?label=Linux%20build).  Interest?

Reply:
That does seem sweet. Will add it to the set of appveyor changes.